src/Razor/src/Microsoft.CodeAnalysis.Remote.Razor/OOPTagHelperResolver.cs

```diff
@@ -208,24 +202,13 @@ protected virtual ImmutableArray<Checksum> ProduceChecksumsFromDelta(ProjectId p
         {
             // Not a delta based response, we should treat it as a "refresh"
             checksums = ImmutableArray<Checksum>.Empty;
-            fromCache = false;
         }
 
         if (deltaResult.ResultId != lastResultId)
         {
             // New results, lets build a coherent TagHelper collection and then cache it
             checksums = deltaResult.Apply(checksums);
             _resultCache.Set(projectId, deltaResult.ResultId, checksums);
-            fromCache = false;
-        }
-
-        stopWatch.Stop();
-        if (fromCache)
-        {
-            _telemetryReporter.ReportEvent(
-                "taghelpers.fromcache",
```

Comment:
Do we need to remove https://github.com/dotnet/razor/blob/2c944fac3b2652c8c7df052d2321b181015e4273/src/Razor/src/Microsoft.AspNetCore.Razor.ProjectEngineHost/CompilationTagHelperResolver.cs#L79 as well?

Reply:
I certainly haven't used it, but  I could see that being useful if someone wants to know which tag helper resolvers are slowest. The data at least isn't completely one-sided :)